docs/sphinx/about/whats-new.txt

```diff
@@ -1,13 +1,14 @@
 Version history
 ===============
 
-5.2.0-m4 (2016 June 24)
+5.2.0-m5 (2016 July 7)
 -----------------------
 
 Top-level Bio-Formats API changes:
 
 * Java 1.7 is now the minimum supported version
 * the native-lib-loader dependency has been bumped to version 2.1.4
+* the xalan dependency has been bumped to version 2.7.2
```

Comment:
@ctrueden: thanks for the quick answer. As @mtbc said, Java 6 support is dropped in Bio-Formats 5.2.0. I will update the release instructions to use the correct update site and we can open a PR directly in `pom-scijava`.

Does this mean Bio-Formats 5.2.x will be conditionally available in Fiji depending on the activation of the Java-8 update site and is there any way to mark this condition if so? /cc @hflynn